contracts/Rewards/RewardsDistributor.sol

```diff
@@ -256,9 +256,7 @@ contract RewardsDistributor is ExponentialNoError, Ownable2StepUpgradeable {
      * @param vTokens The list of markets to claim REWARD TOKEN in
      */
     function claimRewardToken(address holder, VToken[] memory vTokens) public {
-        address[] memory holders = new address[](1);
-        holders[0] = holder;
-        _claimRewardToken(holders, vTokens);
+        _claimRewardToken(holder, vTokens);
```

Comment:
Do we need the internal function? Would it make sense just move the function content of `_claimRewardToken` to this public function? We are not invoking `_claimRewardToken` from any other place

Reply:
cool, will move.

Reply:
[FIXED](https://github.com/VenusProtocol/isolated-pools/pull/163/commits/b3d53030012993270beaf7fa074763c64e5f7645)